CONTRIBUTING.md

```diff
@@ -22,3 +57,45 @@ tox -e integration   # integration tests
 tox                  # runs 'lint' and 'unit' environments
 ```
 
+
+## Build charm
+
+Build the charm in this git repository using:
+
+```shell
+cd mongodb-operator/
+charmcraft pack
+```
+
+### Deploy
+
+```shell
+# Create a model
+juju add-model development
+
+# Enable DEBUG logging
+juju model-config logging-config="<root>=INFO;unit=DEBUG"
+
+# Deploy the charm
+juju deploy ./mongodb_ubuntu-20.04-amd64.charm
+
+## Code overview
+
+The core implementation of this charm is represented by the [MongodbOperatorCharm](./src/charm.py) class. This class will handle the [core lifecycle events](https://juju.is/docs/sdk/events) associated with the charm:
```

Comment:
Seems like links are not "rendered" in code blocks. We can take this paragraph out from the block so it is formatted.